frontend/src/host_orchestrator/main.go

```diff
@@ -34,18 +34,8 @@ import (
 )
 
 const (
-	DefaultSocketPath     = "/run/cuttlefish/operator"
-	DefaultHttpPort       = "1080"
-	DefaultHttpsPort      = "1443"
-	DefaultTLSCertDir     = "/etc/cuttlefish-common/host_orchestrator/cert"
-	DefaultStaticFilesDir = "static"    // relative path
-	DefaultInterceptDir   = "intercept" // relative path
-	DefaultWebUIUrl       = ""
-
-	defaultAndroidBuildURL          = "https://androidbuildinternal.googleapis.com"
-	defaultCVDBinAndroidBuildID     = "9802869"
-	defaultCVDBinAndroidBuildTarget = "aosp_cf_x86_64_phone-userdebug"
-	defaultCVDArtifactsDir          = "/var/lib/cuttlefish-common"
```

Comment:
Removing these forces all those env variables to be defined if you run the binary directly, such as when you're modifying the source code.
Can we keep the defaults here and remove the duplication from the other file instead?

Reply:
Agree, done.